Cabal/Distribution/Compat/DList.hs

```diff
@@ -0,0 +1,36 @@
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  Distribution.Compat.DList
+-- Copyright   :  (c) Ben Gamari 2015-2019
```

Comment:
Should be 2016 instead of 2019? Though if it was up to me, I'd just ditch copyright headers from all modules in favour of the AUTHORS file (which is referred to from `{Cabal,cabal-install}.cabal`).